src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REPORT_TITLE = "fruit,quantity";
+    private static final String ELEMENT_SEPARATOR = ",";
+    private FruitDao fruitDao;
+    private StringBuilder builder;
```

Comment:
StringBuilder is not a service class. Let's create it inside the method `generate`.